notifications_utils/template.py

```diff
@@ -547,10 +548,20 @@ def __init__(
         self.image_url = image_url
         self.page_count = int(page_count)
 
+    @property
+    def page_numbers(self):
+        for i in range(min(self.page_count, self.max_page_count)):
+            yield i + 1
```

Comment:
I don't understand this function, what does it do?

Reply:
Would it be clearer like this?
```python
last_page_number = min(self.page_count, self.max_page_count)
return range(1, last_page_number)
```

Reply:
yes, much clearer to me, thank you! :)